WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java

```diff
@@ -655,6 +697,18 @@ private void addFinanceRelatedWebParts(String container)
         (new PortalHelper(this)).addWebPart("WNPRC Finance Admin");
     }
 
+    private void addWnprcEHRWebparts()
+    {
+        log("Add WNPRC EHR Webparts.");
+
+        //enable Page Admin Mode
+        new SiteNavBar(getDriver()).enterPageAdminMode();
+
+        (new PortalHelper(this)).removeAllWebParts();
+        (new PortalHelper(this)).addWebPart("WNPRC Electronic Health Record ");
+        (new PortalHelper(this)).addWebPart("wnprcUnits");
```

Comment:
This should leave page admin mode when it's done.

```suggestion
        new PortalHelper(getDriver()).doInAdminMode(ph -> {
            ph.removeAllWebParts();
            ph.addWebPart("WNPRC Electronic Health Record ");
            ph.addWebPart("wnprcUnits");
        });
```